packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -152,7 +152,8 @@ export const filters: Record<string,any> = {
  * sorting, do not try to supply your own.
  */
 export const sortings: Record<PostSortingMode,MongoSelector<DbPost>> = {
-  magic: { score: -1 },
+  // filteredScore is added as a synthetic field by filterSettingsToParams
+  magic: { filteredScore: -1 },
```

Comment:
This doesn't always work because `filterSettingsToParams` doesn't always get called (see All Posts page)

Reply:
Whoops. I thought I checked all the places where "magic" is used by evidently not. Should be fixed now.